terraform/infrastructure/vars.tf

```diff
@@ -43,3 +43,8 @@ variable "use_shared_resources" {
   type    = bool
   default = false
 }
+
+variable "log_retention_period" {
```

Comment:
Is it worth highlighting that the value of this is in days? it might help future us so we don't have to dig deep into the TF code to confirm it.

A few options that sprint to mind are maybe change the var name to `log_retention_days` or maybe add a `description` to the var.

Not a major thing though so happy if this stays as is.

Reply:
feel like its a bit obvious from the fact that the values are super high like 2192 months is crazy 😂

Reply:
:+1: yeah, not a major thing.